lib/yml_api.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../hbw/app/models/hbw/activiti/yml_api'
+module Imprint
+  class YMLAPI < HBW::Activiti::YMLAPI
+    class << self
+      def build
+        new(YAML.load_file(Rails.root.join('config/yml_imprint_api.default.yml')))
```

Comment:
Ok, default config is great, but what about custom config ```yml_imprint_api.yml```. Custom config should override default config.